legend-sdlc-test-utils/src/test/java/org/finos/legend/sdlc/test/junit/TestLegendSDLCTestSuiteBuilder.java

```diff
@@ -27,37 +30,43 @@
 import java.util.Collections;
 import java.util.Enumeration;
 import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.IntStream;
 
 public class TestLegendSDLCTestSuiteBuilder
 {
     @Test
     public void testBuildM2MMappingWithTestsTestSuite() throws Exception
     {
-        testTestSuiteBuilder("legend-sdlc-test-m2m-mapping-model-with-tests", 2, 2, 0, 0);
+        Map<String, Set<String>> expectedTestCasesByTestSuite = Maps.immutable.with(
+                "legend::demo::AB { Specific }", Sets.immutable.with("legend::demo::AB Test #1").castToSet(),
+                "model::domain::inmemm2m::mapping::M2MMapping { Specific }", Sets.immutable.with("model::domain::inmemm2m::mapping::M2MMapping Test #1", "model::domain::inmemm2m::mapping::M2MMapping Test #2").castToSet()).toMap();
```

Comment:
Why create an immutable map only to convert it to a mutable one?
```suggestion
        Map<String, Set<String>> expectedTestCasesByTestSuite = Maps.mutable.with(
                "legend::demo::AB { Specific }", Sets.immutable.with("legend::demo::AB Test #1").castToSet(),
                "model::domain::inmemm2m::mapping::M2MMapping { Specific }", Sets.immutable.with("model::domain::inmemm2m::mapping::M2MMapping Test #1", "model::domain::inmemm2m::mapping::M2MMapping Test #2").castToSet());
```

Reply:
Done